topics/rediscli.md

```diff
@@ -544,47 +547,49 @@ by a Redis instance:
     1460100081.165665 [0 127.0.0.1:51706] "set" "foo" "bar"
     1460100083.053365 [0 127.0.0.1:51707] "get" "foo"
 
+Note that it is possible to use to pipe the output, so you can monitor
+for specific patterns using tools such as `grep`.
+
 ## Monitoring the latency of Redis instances
 
 Redis is often used in contexts where latency is very critical. Latency
 involves multiple moving parts within the application, from the client library
 to the network stack, to the Redis instance itself.
 
-The CLI has multiple facilities in order to study the latency of a Redis
-instance, to understand what's the maximum and average latency, and the
-distribution of latencies in the spectrum.
+The CLI has multiple facilities for studying the latency of a Redis
+instance and understanding the latency's maximum, average and distribution.
 
 The basic latency checking tool is the `--latency` option. Using this
-option Redis runs a loop where the `PING` command is sent to the Redis
+option the CLI runs a loop where the `PING` command is sent to the Redis
 instance, and the time to get a reply is measured. This happens 100
 times per second, and stats are updated in a real time in the console:
 
     $ redis-cli --latency
     min: 0, max: 1, avg: 0.19 (427 samples)
 
-The stats are provided in milliseconds. Usually the average latency of
+The stats are provided in milliseconds. Usually, the average latency of
 a very fast instance tends to be overestimated a bit because of the
 latency due to the kernel scheduler of the system running `redis-cli`
 itself, so the average latency of 0.19 above may easily be 0.01 or less.
-However this is usually not a big problem, since we are interested to see
+However this is usually not a big problem, since we are interested in
 events of a few millisecond or more.
 
 Sometimes it is useful to study how the maximum and average latencies
-evolve during time. For this goal the `--latency-history` option is used:
-it works exactly like `--latency`, but each 15 second (by default) a new
-sampling session is started, starting from zero:
+evolve during time. The `--latency-history` option is used for that
+purpose: it works exactly like `--latency`, but every 15 seconds (by
+default) a new sampling session is started beginning with zero:
```

Comment:
I don't know why I wrote "starting from zero", the right thing should be "starting from scratch". What I mean is that it's a new sampling.